frontend/src/lib/navBar.svelte

```diff
@@ -0,0 +1,28 @@
+<main>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
It should be enough for this to be on root `page.html` .  We don't need to add to every component

Reply:
I can remove it if you insist, but I feel that it would be useful to have on a few different pages, just for easier navigation. I don't plan to have it on every page, but since I would like to use it more than once, I felt it was best to make it its own component. Let me know if you disagree.